node_modules/oae-principals/tests/data/users-with-password-alias.csv

```diff
@@ -0,0 +1,25 @@
+user-zfdHliPLa, password1, Matthijs, Nicolaas, nicolaas@alias.test.com
+user-plOPperDe, password2, Pareyn, Bert, bert@alias.test.com
```

Comment:
Is there a particular reason the first 2 lines have spaces after the comma and the rest don't have spaces?